bash/standalone/transferXML.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+if [ -z "$1" -o -z "$2" ] ; then
+    echo use "$0 <path_to_input_dir> <path_to_config_file>"
+    exit
+fi
+
+# set path to watch
+DIR="$1"
+# properties 'user' and 'api_endpoint'
+CONFIG_FILE="$2"
+
+source $CONFIG_FILE
+
+# Key 'user' should be included in credentials file
+if [ -z "$user" -o -z "$api_endpoint" ] ; then
+    echo "Not all required config keys are present: 'user' and 'api_endpoint'"
+    exit 1
+fi
+
+LAST_ADDED_FILE="last_added_file.txt"
+
+touch $LAST_ADDED_FILE
+
+last_filename=$(<$LAST_ADDED_FILE)
+newer_command=""
+
+if test -f "$last_filename"; then
+    newer_command="-newer $last_filename "
+    echo "file exists"
+else
+    newer_command="-mmin -60 "
```

Comment:
Is die frequentie besproken ? Want eens per uur is misschien wat weinig. Ligt een beetje aan hoeveel het er zijn. Als het er veel zijn dan wordt dat zo'n piek elk uur op het systeem.

Reply:
Dit is nog niet besproken. Wat mij betreft kan het ook vaker. Elke 5 minuten?